typescript/agentkit/src/wallet-providers/solanaKeypairWalletProvider.ts

```diff
@@ -84,6 +88,35 @@ export class SolanaKeypairWalletProvider extends SvmWalletProvider {
     }
   }
 
+  /**
+   * Create a new SolanaKeypairWalletProvider from an SVM networkId and a keypair
+   *
+   * @param networkId - The SVM networkId
+   * @param keypair - Either a Uint8Array or a base58 encoded string representing a 32-byte secret key
+   * @returns The new SolanaKeypairWalletProvider
+   */
+  static async fromNetwork<T extends SolanaKeypairWalletProvider>(
```

Comment:
Given how extremely rate limited the Solana public RPC Nodes are, I think any functional Solana app requires a dedicated RPC Node, so an entry point without an RPC URL will not be very useful. By extension, an entry point where a developer species a network (`mainnet-beta`) will not be very useful, since the network can be inferred by querying the genesis hash from the RPC.

Reply:
I understand and agree. 

However, `fromRpcUrl` and even `fromConnection` both exist, so users who have their own dedicated RPC nodes/connections can use one of those options

Reply:
I believe John's thought process is that the example and quickstart should give people a easy way to dip their toes in, without friction. Any serious hackers will get their own dedicated RPC urls eventually, but to get them from 0-1 in a hackathon setting, having defaults (even if rate limited) offers a good experience